### PR TITLE
Added installation steps via helm charts.

### PR DESCRIPTION
**Pull request by gyliu513**
Fixed https://github.com/istio/istio.github.io/issues/412

### CODE REVIEW

**Comment by gyliu513** [comment]
I signed it!

**Comment by gyliu513** [comment]
Thanks @ldemailly , how about make this on top of the installation page, so  that if people want a simple installation steps, they can just goto helm without reading the whole manual installation steps.

**Comment by ldemailly** [comment]
The page you want to link https://github.com/kubernetes/charts/tree/master/incubator/istio
says:
screen shot 2017-07-16 at 1 35 18 am
with "here" being https://istio.io/docs/tasks/installing-istio.html#installation-steps which is this page, thus we would create a loop with your change

Can you first change https://github.com/kubernetes/charts/tree/master/incubator/istio to be self contained (and reference this page as "manual steps" if you want) ?

or again we can add a new section but not at the top - it'll show up in table of content box:

screen shot 2017-07-16 at 1 37 39 am

**Comment by gyliu513** [comment]
Posted a PR in to fix istio helm chart https://github.com/kubernetes/charts/pull/1496 @ldemailly

**Comment by ldemailly** [comment]
Please remove the separate header and instead add the single paragraph:
```
You can use the [istio helm chart](https://github.com/kubernetes/charts/tree/master/incubator/istio) to install, or follow the steps below:
```

**Comment by ldemailly** [comment]
ping @gyliu513

**Comment by ldemailly** [comment]
@gyliu513 thanks for make some of the changes, though I'm not exactly in love with the "manual install" part (should probably be "manual installation", install is a verb (or a colloquial noun/abbrev)

I'll defer to @geeknoid to PTAL

**Comment by ldemailly** [comment]
ps: avoid doing force pushes as it erases the history in github

File: _docs/tasks/installing-istio.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ This page shows how to install and configure Istio in a Kubernetes cluster.
 * If you previously installed Istio on this cluster, please uninstall first by following the
   [uninstalling]({{home}}/docs/tasks/installing-istio.html#uninstalling) steps at the end of this page.
 
-## Installation steps
+You can use the [Istio Helm chart](https://github.com/kubernetes/charts/tree/master/incubator/istio) to install, or follow the steps below for manual installation:
+
+## Manual installation steps
```
Comment (geeknoid) [inline _docs/tasks/installing-istio.md]: I would move the paragraph above inside of the # Installation steps header, and leave the header name unchanged.

There should be steps in the Uninstalling section regarding Helm charts too.

Comment (gyliu513) [inline _docs/tasks/installing-istio.md]: Done, thanks @geeknoid

**Comment by gyliu513** [comment]
Removed the update to istio auth, thanks @ldemailly

**Pull request merged**